app/scripts/directives/simple-slider.js

```diff
@@ -1,20 +1,50 @@
 angular.module('angularSimpleSlider')
-  .directive('simpleSlider', ['SimpleSliderService', function (SimpleSliderService) {
+  .directive('simpleSlider', ['SimpleSliderService', '$timeout', function (SimpleSliderService, $timeout) {
 
     'use strict';
 
     return {
 
-      restrict: 'E',
-      scope: {},
+      restrict: 'AE',
+      scope: {
+        onChange: '&',
+        current: '=?currentSlide',
+        slider: '=?sliderInstance'
+      },
 
       link: function postLink(scope, element, attrs) {
+        var options = attrs, watcher;
```

Comment:
maybe rename `watcher` to a more informative name? I find `disposeWatcher` to be a better name in this case, I had to go to the angular documentation to remember what `$scope.watch` would return :stuck_out_tongue_closed_eyes: